extras/DAC_calibration/Calibration.rst

```diff
@@ -0,0 +1,321 @@
+Calibrate QICK units
+====================
+
+QICK (and therefore Qibosoq) uses arbitrary units to set the amplitudes
+of its pulses. It is not easy to directly connect a specific value of
+amplitude to a physical one in volt or dBm because this depends on two
```

Comment:
Not sure about the meaning of expressing a tension in `dBm`. That should be the attenuation, but on top of somehting - and usually that something is the DAC output, so you will need sooner or later a value in Volt.

Reply:
Sorry, I confused `dB` and `dBm`. However, why you should measure a pulse in W? Are you assuming the output current to be fixed? Doesn't it also depend on the load?

Reply:
dBm are the standard in this type of measurements, I think that's the case beacuse the load is always fixed at 50 Ohm